public/electron/fuse/index.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path');
```

Comment:
is not better handle this as a helper? I mean, we can call a function or a promise to do this on demand when the user need it?

Reply:
Yeah, wrapping this in a helper would be good. Like I said earlier, I wasn't sure how best it would integrated with the whole app, I just did this so you can see the installation logic. Where it would be called would depend on the frontend team.